llvm/test/CodeGen/AArch64/arm64-addrmode.ll

```diff
@@ -209,3 +209,88 @@ define void @t17(i64 %a) {
   %3 = load volatile i64, ptr %2, align 8
   ret void
 }
+
+; https://gcc.godbolt.org/z/ErhhdxMv3
```

Comment:
It's probably best to remove the godbolt links from the tests.

Reply:
Done